app/components/variable-height-textarea.js

```diff
@@ -5,18 +5,28 @@ export default Ember.TextArea.extend({
   attributeBindings: ["disabled"],
   disabled: false,
 
+   didDestroyElement: function () {
+     Ember.$('.liquid-container').css({
```

Comment:
Should we add a check if liquid container exists or not?

Reply:
It's always there. But I'll add a check if you want.